### PR TITLE
Add links to Contour and Harbor

### PR DESCRIPTION
**Pull request by AcidLeroy**
- Added basic documentation, some logos, and the links to each respective supervisor service in Artifactory.

Signed-off-by: Cody W. Eilar <ecody@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,8 +65,55 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
...
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
+
+
+**Versions:**
+- Download latest version: [Harbor v2.5.3](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/harbor/v2.5.3/harbor.yml) 
+
+**Harbor Sample values.yaml**
+- Download the latest version: [values.yaml sample
+  v2.5.3](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/harbor/v2.5.3/harbor-data-values.yml)
+
```
Comment (fstrudel) [inline README.md:96]: Can we explain the main values though?

File: README.md [hunk 2]
```diff
@@ -65,8 +65,55 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
...
-- Download latest version: [CA Cluster - v0.0.1]()  
+- Download latest version: [CA Cluster - v0.0.1](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/ca-clusterissuer/v0.0.1/ca-clusterissuer.yml)  
 
 **CA Cluster Issuer Sample values.yaml**
 - Download latest sample values.yaml: [values.yaml for
-  v0.0.1](caclusterissuer/values.yml)
+  v0.0.1](https://vmwaresaas.jfrog.io/ui/native/supervisor-services/ca-clusterissuer/v0.0.1/ca-clusterissuer-data-values.yml)
+
+
+## Cloud Native Registry
```
Comment (fstrudel) [inline README.md:75]: Can we indicate somewhere that you need to install contour for harbor to work? (actually i have not checked the linked doc if that is clear)

File: README.md [hunk 3]
```diff
@@ -65,8 +65,55 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
...
+
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
+
+
+**Versions:**
+- Download latest version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/contour/v1.18.2/contour.yml) 
+
+**Contour Sample values.yaml**
+- Download the latest version: [values.yaml sample
+  v1.18.2](https://vmwaresaas.jfrog.io/artifactory/supervisor-services/contour/v1.18.2/contour-data-values.yml)
```
Comment (fstrudel) [inline README.md:118]: Same. Add some info on main key parameters?

**Review by fstrudel — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -65,8 +65,56 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
...
+  <img src="harbor-logo.png" width="250" title="Harbor Logo">
+</p>
+</br>
+Harbor is an open source trusted cloud native registry project that stores, signs, and scans content. Harbor extends the open source Docker Distribution by adding the functionalities usually required by users such as security, identity and management. Having a registry closer to the build and run environment can improve the image transfer efficiency. Harbor supports replication of images between registries, and also offers advanced security features such as user management, access control and activity auditing.
+</br>
+</br>
+
+- The [contour package](#contour) is a prerequisite for Harbor, so that must be installed
+  first. 
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
```
Comment (YuxinZhou) [inline README.md:89]: https://docs-staging.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-EB11DFDE-6ED4-4864-9E60-669F50D64397.html 

Maybe we could refer to this page instead. This is more detailed and specific to harbor and contour.

File: README.md (hunk 1 above)
Comment (AcidLeroy) [reply README.md]: I added a page that describes these.

File: harbor/README.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Harbor Configuration 
+
+Below is table that highlights the required fields for the Harbor data values file. 
+
+|  Property	                                                 | Value	              | Description    |                                                                                         
+| ---------------------------------------------------------- | -------------------- | -------------- |                                                                                        
+| hostname	                                                 | FQDN	                | The FQDN that you have designated to access the Harbor UI and for referencing the registry in client applications.|
```
Comment (douglas-c) [inline harbor/README.md:7]: Might be good to add a note here: The domain should be configured in an external DNS server such that it resolves to the Envoy Service IP created by Contour.

File: README.md [hunk 6]
```diff
@@ -65,8 +65,56 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
...
+</br>
+
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-4843E6C6-747E-43B1-AC55-8F02299CC10E.html).
+
+
+**Versions:**
+- Download latest version: [Contour v1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour.yml) 
+
+**Contour Sample values.yaml**
+- Download the latest version: [values.yaml samplev1.18.2](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=contour/v1.18.2/contour-data-values.yml). These values can be used _as-is_ and require no configuration changes.
```
Comment (douglas-c) [inline README.md:119]: Link text looks wrong, I would expect "values for v1.18.2" or the full filename (which ends in .yml not .yaml).

File: README.md [hunk 7]
```diff
@@ -65,8 +65,56 @@ ClusterIssuers are Kubernetes resources that represent certificate authorities (
...
+- The [contour package](#contour) is a prerequisite for Harbor, so that must be installed
+  first. 
+- Service install - Follow steps 1 - 5 in the [documentation](https://docs-staging.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-EB11DFDE-6ED4-4864-9E60-669F50D64397.html).
+
+
+**Versions:**
+- Download latest version: [Harbor v2.5.3](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=harbor/v2.5.3/harbor.yml) 
+
+**Harbor Sample values.yaml**
+- Download the latest version: [values.yaml sample
```
Comment (douglas-c) [inline README.md:96]: Link text looks wrong, I would expect "values for v2.5.3" or the full filename.

**Pull request merged**